src/capybara.js

```diff
@@ -285,13 +297,9 @@ Capybara = {
       }
 
       if (!node.readOnly) {
-        this.focus(index);
```

Comment:
This may change the order of events triggered from the node, since the focus would now happen after the value was cleared instead of before. If possible, I'd prefer to keep the existing order to avoid weird backwards compatibility issues.